### PR TITLE
fix redis session standalone stream ssl context

### PR DESCRIPTION
**Pull request by patricio-dorantes**
Configure redis standalone session w/tls.

### CODE REVIEW

File: redis_session.c [hunk 1]
```diff
@@ -451,6 +451,7 @@ PS_OPEN_FUNC(redis)
                 REDIS_CONF_AUTH_STATIC(ht, "auth", &user, &pass);
 
                 zval_dtor(&params);
+                context = REDIS_HASH_STR_FIND_TYPE_STATIC(ht, "stream", IS_ARRAY);
```
Comment (yatsukhnenko) [inline redis_session.c:453]: You use `ht` right after free it.

File: redis_session.c [hunk 2]
```diff
@@ -490,6 +491,10 @@ PS_OPEN_FUNC(redis)
                                            persistent, persistent_id ? ZSTR_VAL(persistent_id) : NULL,
                                            retry_interval);
 
+            if (context  != NULL) {
```
Comment (yatsukhnenko) [inline redis_session.c:495]: You need to initialise `context` with `NULL` before parsing params.

**Review by yatsukhnenko — CHANGES_REQUESTED** [review]

File: redis_session.c (hunk 1 above)
Comment (patricio-dorantes) [reply redis_session.c:453]: 😅 got it.

File: redis_session.c (hunk 2 above)
Comment (patricio-dorantes) [reply redis_session.c:495]: done :). Thank you for your patience.

Comment (yatsukhnenko) [reply redis_session.c:495]: This is not exactly what I asked but ok :) just add `context = NULL` after `redis_sock_set_stream_context`

**Review by yatsukhnenko — CHANGES_REQUESTED** [review]

File: redis_session.c (hunk 2 above)
Comment (patricio-dorantes) [reply redis_session.c:495]: 😂😅

**Pull request merged**